oci/spec_opts_unix.go

```diff
@@ -36,10 +36,10 @@ func WithHostDevices(_ context.Context, _ Client, _ *containers.Container, s *Sp
 	return nil
 }
 
-// WithDevices recursively adds devices from the passed in path and associated cgroup rules for that device.
+// WithDevices recursively adds devices from the passed in path.
 // If devicePath is a dir it traverses the dir to add all devices in that dir.
 // If devicePath is not a dir, it attempts to add the single device.
-func WithDevices(devicePath, containerPath, permissions string) SpecOpts {
+func WithDevices(devicePath, containerPath, permissions, ownership string) SpecOpts {
```

Comment:
https://github.com/kubernetes/kubernetes/issues/92211 pls read this discussion.. this is a prototyping exercise..

Reply:
@kzys great questions btw, will queue these up for our next cod call I think :-)

Reply:
cri-o does not need this syntax because of the way UID/GID can be set there. It used to be the same for containerd but 7776e5ef2ad42cc6e739241db677049c6b083ab1 made it harder.